templates/services/merge/execution/reth.tmpl

```diff
@@ -0,0 +1,44 @@
+{{/* reth.tmpl */}}
+{{ define "execution" }}
+  execution:
+    stop_grace_period: 30m
+    container_name: sedge-execution-client{{if .ContainerTag}}-{{.ContainerTag}}{{end}}
+    restart: unless-stopped
+    image: ${EC_IMAGE_VERSION}
+    networks:
+      - sedge
+    volumes:
+      - ${EC_DATA_DIR}:/root/.local/share/reth/mainnet
+      - ${EC_JWT_SECRET_PATH}:/root/jwt:ro
```

Comment:
Some typo in there - with such config it crashes the process with error:
Error: failed to read from "/root/jwt/jwt.hex": Not a directory (os error 20)

Probably mapped path is invalid with this ":ro" at the end

Reply:
In [https://github.com/paradigmxyz/reth](reth) etc/docker-compose.yml these are the directories written:

- In volumes: 
`- ./jwttoken:/root/jwt:ro`

- In Command:
`--authrpc.jwtsecret /root/jwt/jwt.hex`

Reply:
What I did and it works is:
- ./jwttoken:/root/jwt
--authrpc.jwtsecret /root/jwt